src/System.Private.ServiceModel/tools/setupfiles/sync-pr.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+show_banner()
+{
+    echo ""
+    echo "WCF Remote Github Repo sync script"
+}
+
+show_usage() 
+{
+    echo "    A URL must be specified for the pull request synchronization URL"
+    echo "    Usage: $0 [sync-url]"
+    echo "    sync-url - URL on remote server for PR synchronization"
+    echo ""
+    echo "    Example:  $0 http://wcfci-sync-server/repo/sync.ashx"
+}
+
+run_request() 
+{
+    __sync_url=$1
+    __ghprbPullId=$2
+
+    __request_uri=${__sync_url}?pr=${__ghprbPullId}
+
+    echo "Making a call to '${__request_uri}'" 
+
+    if [ ! -e "$__nugetpath" ]; then
+        which curl wget > /dev/null 2> /dev/null
```

Comment:
Oops, copy+paste error on this one. Line 27 should go away. 
This is to figure out whether or not we have curl or wget installed.